### ISSUE TITLE
kotlin.UninitializedPropertyAccessException: lateinit property lookup has not been initialized

### ISSUE THREAD

**Issue by m-aciek** [tonybaloney/pycharm-security#3]
```
kotlin.UninitializedPropertyAccessException: lateinit property lookup has not been initialized
	at security.packaging.SafetyDbChecker.hasMatch(SafetyDbChecker.kt:42)
	at security.packaging.PyPackageSecurityScan.checkPackages(PyPackageSecurityScan.kt:32)
	at security.packaging.PythonPackageVulnerabilityStartupTask$runActivity$1.run(PythonPackageVulnerabilityStartupTask.kt:22)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:888)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:163)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:585)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:531)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:59)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:150)
	at com.intellij.openapi.progress.impl.CoreProgressManager$4.lambda$run$0(CoreProgressManager.java:402)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:221)
	at com.intellij.openapi.progress.impl.CoreProgressManager$4.run(CoreProgressManager.java:402)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:238)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```
Python Security 1.0.5

**Comment by jugmac00** [comment]
@tonybaloney When you fix this bug, please reference the pr/commit and briefly explain the problem. Thank you!

**Comment by tonybaloney** [comment]
Fixed in 1.0.7. Sorry about that! I refactored it whilst setting up unit tests. 
At the moment I can't get the unit tests working, so I've missed a few really silly mistakes like this

**Comment by m-aciek** [comment]
Thank you!
